src/ciflow-bot.ts

```diff
@@ -5,26 +5,48 @@ import {CachedIssueTracker} from './utils';
 const ciflowCommentStart = '<!-- ciflow-comment-start -->';
 const ciflowCommentEnd = '<!-- ciflow-comment-end -->';
 
-export function parseCIFlowIssue(rawText: string): object {
+interface IUserConfig {
+  githubHandler: string;
```

Comment:
```suggestion
  login: string;
```

can we call this `login` to be consistent with the GitHub API? or at least `githubHandle` since it's just a string